Fix customer reported issues with documentation (#1281)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/parameters.md

- `false` [default value] The user is able to turn switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.
- `true`  The user is prevented from turning off the switch.

**Warning**

> You must also include `auto_connect` with at least a value of 0 on new deployments or cleints will be deployed in the off state without a way for users to manually enable.

Reviewer: ```suggestion
<Aside>
On new deployments, you must also include the `auto_connect` parameter with at least a value of 0. This will prevent clients from being deployed in the off state without a way for users to manually enable them.
</Aside>
```

<Aside> 

>   This parameter replaces the old enabled property which can no longer be used in conjunction with the new switch_locked and auto_connect. If you want to use switch_locked and/or auto_connect you must stop using enabled.

Reviewer: ```suggestion
  This parameter replaces the old `enabled` property, which can no longer be used in conjunction with the new `switch_locked` and `auto_connect`. If you want to use these parameters, you must remove `enabled`.
```

- `1-1440`  Turn switch back on automatically after the specified number of minutes.

<Aside> 

>   This parameter replaces the old enabled property which can no longer be used in conjunction with the new switch_locked and auto_connect. If you want to use switch_locked and/or auto_connect you must stop using enabled.

Reviewer: ```suggestion
  This parameter replaces the old `enabled` property, which can no longer be used in conjunction with the new `switch_locked` and `auto_connect`. If you want to use these parameters, you must remove `enabled`.
```
